atomium/includes/common.inc

```diff
@@ -496,13 +496,14 @@ function atomium_menu_get_active_breadcrumb() {
 
     foreach ($active_trail as $parent) {
       $breadcrumb[] = array(
-        '#theme' => array('link'),
+        '#theme' => array('link__breadcrumb'),
```

Comment:
why we do not use _atomium_extend_theme_hook function?

Reply:
Thanks, fixed.